convert2rhel/subscription.py

```diff
@@ -413,3 +415,36 @@ def check_needed_repos_availability(repo_ids_needed):
             all_repos_avail = False
     if all_repos_avail:
         loggerinst.info("Needed RHEL repos are available.")
+
+
+def download_subscription_manager_packages():
+    loggerinst = logging.getLogger(__name__)
+
+    _CENTOS_CONTRIB_REPO = '[contrib]\n' \
+                           'name=CentOS-$releasever - Contrib\n' \
+                           'mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib\n' \
+                           'gpgcheck=0\n' \
+                           'enabled=1\n' \
+                           'protect=0\n'
+
+    _TEMP_REPOFILE_DIR = '/tmp/convert2rhel/centos_contrib.repo'
+
+    version = utils.parse_version_number(system_info.system_release_file_content)
+
+    if int(version['major']) <= 6 and (version['minor'] is None or ('minor' in version and int(version['minor']) <= 6)):
```

Comment:
Since they're now ints we don't need to convert it on the fly

```suggestion
    if version['major'] <= 6 and (version['minor'] is None or ('minor' in version and version['minor'] <= 6)):
```